Documentation/ceph-rbd-mirror-crd.md

```diff
@@ -40,8 +40,64 @@ If any setting is unspecified, a suitable default will be used automatically.
 ### RBDMirror Settings
 
 * `count`: The number of rbd mirror instance to run.
+* `peers`: to configure mirroring peers
+  * `secretNames`:  a list of peers to connect to. Currently (Ceph Octopus release) **only a single** peer is supported where a peer represents a Ceph cluster.
+  However, if you want to enable mirroring of multiple pools, you would have to have **one Secret per pool**, but the token (the peer identity) must be the same.
 * `placement`: The rbd mirror pods can be given standard Kubernetes placement restrictions with `nodeAffinity`, `tolerations`, `podAffinity`, and `podAntiAffinity` similar to placement defined for daemons configured by the [cluster CRD](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/cluster.yaml).
 * `annotations`: Key value pair list of annotations to add.
 * `labels`: Key value pair list of labels to add.
 * `resources`: The resource requirements for the rbd mirror pods.
 * `priorityClassName`: The priority class to set on the rbd mirror pods.
+
+### Configuring mirroring peers
+
+On an external site you want to mirror with, you need to create a bootstrap peer token.
+The token will be used by one site to **pull** images from the other site.
+The following assumes the name of the pool is "test" and the site name "europe" (just like the region), so we will be pulling images from this site:
+
+```console
+external-cluster-console# rbd mirror pool peer bootstrap create test --site-name europe
+```
+
+For more details, refer to the official rbd mirror documentation on [how to create a bootstrap peer](https://docs.ceph.com/docs/master/rbd/rbd-mirroring/#bootstrap-peers).
+
+When the peer token is available, you need to create a Kubernetes Secret.
+Our `europe-cluster-peer-pool-test` will have to be created manually, like so:
+
+```console
+kubectl -n rook-ceph create secret generic "europe-cluster-peer-pool-test" \
+--from-literal=token=eyJmc2lkIjoiYzZiMDg3ZjItNzgyOS00ZGJiLWJjZmMtNTNkYzM0ZTBiMzVkIiwiY2xpZW50X2lkIjoicmJkLW1pcnJvci1wZWVyIiwia2V5IjoiQVFBV1lsWmZVQ1Q2RGhBQVBtVnAwbGtubDA5YVZWS3lyRVV1NEE9PSIsIm1vbl9ob3N0IjoiW3YyOjE5Mi4xNjguMTExLjEwOjMzMDAsdjE6MTkyLjE2OC4xMTEuMTA6Njc4OV0sW3YyOjE5Mi4xNjguMTExLjEyOjMzMDAsdjE6MTkyLjE2OC4xMTEuMTI6Njc4OV0sW3YyOjE5Mi4xNjguMTExLjExOjMzMDAsdjE6MTkyLjE2OC4xMTEuMTE6Njc4OV0ifQ== \
+--from-literal=pool=test
+```
+
+Rook will read both `token` and `pool` keys of the Data content of the Secret.
+Rook also accepts the `destination` key, which specifies the mirroring direction.
+It defaults to rx-tx for bidirectional mirroring, but can also be set to rx-only for unidirectional mirroring.
+
+You can now inject the rbdmirror CR:
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephRBDMirror
+metadata:
+  name: my-rbd-mirror
+  namespace: rook-ceph
+spec:
+  count: 1
+  peers:
```

Comment:
Will we ever have any other settings under the `peers` besides the secret names? If not, what about this?
```
peerSecretNames:
- "europe-cluster-peer-pool-test"
```

Reply:
We never know... That's why I kept room for more.